source/dpaa_shutdown_bepa.rst

```diff
@@ -200,7 +204,10 @@ CAPs
 .. |cap1407_doc| replace:: DOC
 .. _cap1407_doc: http://cxc.cfa.harvard.edu/acis/CAPs/CAP1407_dpaa_poweroff_recovery.doc
 
-* CAP 1622 (Update TXINGS Parameter Values)  (``acis_docs/CAPs/CAP1622_TXINGB_SETPARAMS.pdf``) (``acis_docs/CAPs/CAP1622_TXINGB_SETPARAMS.docx``)
+* CAP 1708 (Update TXINGS Parameter Values)  (``acis_docs/CAPs/CAP1708_TXINGB_SETPARAMS.pdf``)
```

Comment:
@gregg140 This is where it looks like there's been a newline added, but I may be wrong.

Reply:
Fixed